### PR TITLE
Add Agent support to CommandStep

### PR DESCRIPTION
**Pull request by johnjcsmith**
Adds support for specifying `agent` on the `CommandStep`

### CODE REVIEW

File: src/builders/CommandStep.test.ts [hunk 1]
```diff
@@ -142,4 +142,16 @@ describe(CommandStep.name, () => {
...
     });
   });
+
+  describe('agents', () => {
+    test('undefined by default', () => {
+      const step = new CommandStep().command(':').build();
+      expect(step).not.toHaveProperty('agents');
+    });
+
+    test('defined when 2', () => {
```
Comment (jameslnewell) [inline src/builders/CommandStep.test.ts:152]: when 2?

File: src/builders/CommandStep.ts [hunk 2]
```diff
@@ -8,6 +8,7 @@ import {SkipBuilder, SkipHelper} from './helpers/skip';
 import {PluginBuilder} from './PluginBuilder';
 import {CommandStepSchema, PluginSchema, StepDependsOn} from '../schema';
 import {isPluginBuilder} from './isPluginBuilder';
+import {AgentsHelper} from './helpers/agents';
 
 export class CommandStep
   implements
```
Comment (jameslnewell) [inline src/builders/CommandStep.ts:14]: Can we implement the `AgentsBuilder` too 🙏

Comment (johnjcsmith) [reply src/builders/CommandStep.ts:14]: ✅

File: src/builders/CommandStep.test.ts (hunk 1 above)
Comment (johnjcsmith) [reply src/builders/CommandStep.test.ts]: 🤦

**Review by jameslnewell — APPROVED** [review]

**Pull request merged**